test/domain/notification/getRecentAggregated.spec.ts

```diff
@@ -15,29 +15,45 @@ beforeEach(async () =>
 
 describe('domain/notification/getallAggregated', () =>
 {
-    it('should give all posts for the requester', async () =>
+    it('should give all notifications for the requester2', async () =>
```

Comment:
```suggestion
    it('should give all notifications under normal circumstances', async () =>
```

The specific requester doesn't matter because it's fixture related, and is not a part of what we're testing.